eclipse-collections-code-generator/src/main/resources/test/map/abstractObjectPrimitiveMapKeyValuesViewTestCase.stg

```diff
@@ -757,6 +757,15 @@ public abstract class AbstractObject<name>MapKeyValuesViewTestCase
         Assert.assertEquals(TreeSortedMap.newMapWith(Comparators.reverseNaturalOrder(), "1:<(toStringLiteral.(type))("2")>", "1:<(toStringLiteral.(type))("2")>", "2:<(toStringLiteral.(type))("3")>", "2:<(toStringLiteral.(type))("3")>", "3:<(toStringLiteral.(type))("4")>", "3:<(toStringLiteral.(type))("4")>"), map);
     }
 
+    @Test
+    public void toSortedMap_with_function()
```

Comment:
Please change the test method name to `toSortedMapBy()`.  Same case for all other test case files.

`*_with_*` naming convention is used to test overloaded methods.

Reply:
Fixed.